micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/localizationUtility.js

```
@@ -0,0 +1,30 @@
+export const buildLocalizationMessages = (additionalProperties, moduleName, locale) => {
+    if (!additionalProperties || typeof additionalProperties !== "object") return [];
+  
+    const transformedModuleName = tranformLocModuleName(moduleName).toLowerCase();
+    const messages = [];
+  
+    for (const fieldName in additionalProperties) {
+      if (additionalProperties.hasOwnProperty(fieldName)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)